.github/workflows/lint-markdown.yml

```
@@ -0,0 +1,25 @@
+name: Lint markdown
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened]
+
+jobs:
+  mdlint:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        name: Checkout
+        with:
+          ref: ${{ github.ref }}
+
+      - uses: articulate/actions-markdownlint@v1
+        name: Lint ADRs
+        with:
+          files: 'docs/adrs/*.md'
```

```suggestion
          files: 'docs/**/*.md'
```
This will cover us for any new directories that are added in the future!